README.md

```diff
@@ -50,6 +50,12 @@ NXST_FLOW reply (xid=0x4):
  cookie=0x0, duration=103.842s, table=0, n_packets=0, n_bytes=0, idle_age=103, priority=0 actions=NORMAL
 ```
 
+We have also extended Oko with the CLI command to update the BPF maps of the given BPF program. Once the filter prog
+is loaded, the below command can be used to write entry (key-value pair) to the given BPF program.
+
+```
+ovs-ofctl update-map br0 1 0 key 1 0 0 0 value 1 0 0 0 0 0 0 0 1 0 0 0 0 0 0 0
+```
```

Comment:
Maybe add this line to the above example and change it into an `update-map` operation whose consequences are visible to the user? For example, update the rate limiter such that the effect is visible in the `n_packets` of the first rule. If that's too complex, maybe adding a new example program (simple match based on map content?) would be better. Simpler is better for examples IMHO.

Reply:
I've pushed new commit with a simple 'dropper' example and README updated.

Reply:
Great! Could we merge the two examples? I'd keep only the three first commands from the first example and then would replace the following commands with your example. That way users can try out all actions permitted by Oko in a single scenario.

Reply:
The example BPF program looks great!

Reply:
I've merged them. Is it ok now?

Reply:
I've applied modifications to the example BPF program. Now, I think it looks good.

Reply:
Yep :+1: 

Could you add `bash` to the first line to get syntax highlighting of the comment too?